docs/build-with-powerloom/devnet/getting-started.md

```diff
@@ -116,7 +121,7 @@ This should allow developers to build and experiment with a variety of use cases
 - `NAMESPACE`: The unique key used to identify your project namespace.
 - `POWERLOOM_REPORTING_URL`: The URL for reporting to PowerLoom.
 - `PROST_CHAIN_ID`: Enter the Devnet chain ID
-- `SLOT_ID`: Enter your Devnet Slot ID. [You can find the slotID on the [Devnet dashboard](https://devnet-mint.powerloom.dev/dashboard)]
+- `SLOT_ID`: Enter your Devnet Slot ID. [You can find the slotID by visiting the [Devnet Dashboard](https://devnet-mint.powerloom.dev/dashboard), or on the [Sepolia block explorer](https://sepolia.etherscan.io/) by searching for the transaction hash of your node mint]
```

Comment:
@Seth-Schmidt `slot_id` is repeated two times. Line 104 and 124

Reply:
Removed the extra slot_id: [4a04500](https://github.com/PowerLoom/docs/pull/26/commits/4a0450085f95b2c5056f37e6cda2fdf1c29cebb1)